test/forked/rails_rake_full_stack_test.rb

```diff
@@ -17,4 +17,16 @@ class RailsRakeFullStackTest < Minitest::Test
     pundit_coverage = store.coverage[pundit_file]
     assert_nil pundit_coverage
   end
+
+  test "ignored rake tasks don't add coverage" do
+    store.instance_variable_set(:@redis_namespace, 'coverband_test')
```

Comment:
Can this be moved to test_helper possibly? Also, does this work since we are shelling out on line 24?

Reply:
it does, it is weird... basically, we shell out and the shelled out code used the dummy rails `config/coverband.rb` which sets the namespace in the shell... this line sets the namespace for the code not shelled out where we call `store` directly to check the data... If we don't set it here we are checking against a different redis namespace.

Reply:
Ok. Should we just globally set our namespace to coverband_test in test_helper?

Reply:
I had tried that, not exactly sure why it wasn't always getting picked up... See the settings in the helper...

https://github.com/danmayer/coverband/blob/master/test/test_helper.rb#L32

Any ideas why this would be in a state where the store doesn't have the correct namespace @kbaum ?